deploy/iso/minikube-iso/arch/aarch64/package/containerd-bin-aarch64/config.toml

```diff
@@ -40,8 +40,7 @@ oom_score = 0
     [plugins."io.containerd.grpc.v1.cri".containerd]
       discard_unpacked_layers = true
       snapshotter = "overlayfs"
-      [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
-        runtime_type = "io.containerd.runc.v2"
+      default_runtime_name = "runc"
```

Comment:
Should this also be removed?

https://github.com/kubernetes/minikube/blob/4f3d0da802c7514e57551f42ba6fb9f0abc3d4b0/deploy/iso/minikube-iso/arch/aarch64/package/containerd-bin-aarch64/config.toml.default#L77

Reply:
similar to the comment below: we'll remove it as agreed